### PR TITLE
Several corrections.  Mostly .md to .html

### PR DESCRIPTION
**Pull request by lkurz**

### CODE REVIEW

File: documentation/hdf5-docs/advanced_topics_list.md [hunk 1]
```diff
@@ -9,18 +9,18 @@ redirect_from:
...
-| [HDF5 Data Flow Pipeline for H5Dread](advanced_topics/data_flow_pline_H5Dread.md) | Describes data flow when reading raw data from an HDF5 dataset |
+| [HDF5 Data Flow Pipeline for H5Dread](advanced_topics/data_flow_pline_H5Dread.html) | Describes data flow when reading raw data from an HDF5 dataset |
 | [HDF5 Metadata](advanced_topics/) | Provides a comprehensive overview of the types of metadata used in HDF5 |
-| [Introduction to Single-Writer_Multiple-Reader (SWMR)](advanced_topics/intro_SWMR.md) | Enables writing to a file while multiple readers are accessing it |
-| [Introduction to the Virtual Dataset - VDS](advanced_topics/intro_VDS.md) | Describes how to present data stored in several HDF5 datasets and files as a single dataset and access the data using HDF5 APIs |
+| [Introduction to Single-Writer_Multiple-Reader (SWMR)](advanced_topics/intro_SWMR.html) | Enables writing to a file while multiple readers are accessing it |
+| [Introduction to the Virtual Dataset - VDS](advanced_topics/intro_VDS.) | Describes how to present data stored in several HDF5 datasets and files as a single dataset and access the data using HDF5 APIs |
```
Comment (lkurz) [inline documentation/hdf5-docs/advanced_topics_list.md:23]: Missing html extension.

**Review by byrnHDF — APPROVED** [review]

**Comment by brtnfld** [comment]
What is the relationship between the html and md files? Are the md files with the same name as the HTML files still needed?

**Comment by lkurz** [comment]
>>What is the relationship between the html and md files? Are the md files with the same name as the HTML files still needed?
                .md files are the source code.
.html files are the result of a jekyll build
They share the same file name.
                Links to internal files (relative links) need to use .html
External links, such as to a markdown file from a github repo need to be .md

From: Scot Breitenfeld ***@***.***>
Reply-To: "HDFGroup/HDFGroup.github.io" ***@***.***>
Date: Wednesday, August 28, 2024 at 4:42 PM
To: "HDFGroup/HDFGroup.github.io" ***@***.***>
Cc: Les Kurz ***@***.***>, Assign ***@***.***>
Subject: Re: [HDFGroup/HDFGroup.github.io] Several corrections. Mostly .md to .html (PR #300)


What is the relationship between the html and md files? Are the md files with the same name as the HTML files still needed?

**Review by gheber — APPROVED** [review]
All good. We should capture this piece of information somewhere in this repo. How about REMADME.md?

**Pull request merged**